app-misc/vcontrold/vcontrold-0.98.11.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+DESCRIPTION="Daemon for communication with Viessmann Vito heatings"
+HOMEPAGE="https://github.com/openv/vcontrold/"
+SRC_URI="https://github.com/openv/vcontrold/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+IUSE="+man +vclient vsim"
+
+RDEPEND="dev-libs/libxml2:2"
+DEPEND="${RDEPEND}
+	man? ( dev-python/docutils )"
```

Comment:
This most likely belongs to BDEPEND. I also fear you may have to use a python eclass to get it working properly, 
https://projects.gentoo.org/python/guide/any.html
https://projects.gentoo.org/python/guide/any.html#dependencies